ui/lib/pki/addon/components/pki-key-parameters.hbs

```diff
@@ -49,7 +49,7 @@
                         data-test-input={{attr.name}}
                       >
                         {{#each (path-or-array attr.options.possibleValues @model) as |val|}}
-                          <option selected={{eq (get @model this.valuePath) (or val.value val)}} value={{or val.value val}}>
```

Comment:
caused an issue with the default time of 0 selection. need to just return the value.